compiler/build.gradle

```diff
@@ -13,6 +13,8 @@ repositories {
 
 dependencies {
 
+    implementation androidApi
```

Comment:
This is required as `ViewModelProviders` now references `android.app.Activity` and the annotation processor does not recognize it otherwise.